transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImpl.kt

```diff
@@ -386,15 +385,8 @@ internal class MessagingClientImpl(
                 if (structuredMessage.isOutbound()) {
                     structuredMessage.events.forEach {
                         if (it.isDisconnectionEvent()) {
-                            if (deploymentConfig.isConversationDisconnectEnabled()) {
```

Comment:
Why this is not needed anymore?
The BE forces session status (like readOnly) no matter if the feature is enabled on the console or if we got it, it must be enabled.

Reply:
For the Conversation.Disconnect flow we should treat BE (Shyrka) as an only source of truth. DeploymentConfig might be changed at any point, but we still have to respect whatever Shyrka sends us. 
For example: 
1. Admin has disabled ConversationDisconnect feature. 
2. Transport has fetched latest DeploymentConfig.
3. Shyrka still treat this session like ConversationDisconnect was enabled.
4. Shyrka sends us Event.ConversationDisconnect with/without readonly.
5. TransportSDK ignore this event/session state change.
6. From shyrka perspective session should be disconnected, but from Transport it is still active. 

By removing this checks we respect whatever Shyrka sends us without checking/falling back to DeploymentConfig.